### ISSUE TITLE
Python3 API: Full list of API options

### ISSUE THREAD

**Issue by Kentoseth** [lxc/lxc#1033]
I am not sure if this is due to a lack of documentation, but after looking here:

[github-python-lxc](https://github.com/lxc/lxc/tree/master/src/python-lxc)
[Ubuntu LXC](https://help.ubuntu.com/lts/serverguide/lxc.html#python-lxc)
[LXC Docs](https://linuxcontainers.org/lxc/documentation/)

I believe the full list of available API commands for Python3 is not clearly or well-documented.

Does anybody have an alternate reference I can source?

I tried looking at the MAN page on the LXC site as well, but I don't think that specifically addresses the full list of API options (for Python3).

**Comment by Kentoseth** [comment]
Okay, looks like I missed this when perusing the documentation:

> Only symbols listed in lxccontainer.h are part of the API, everything else is internal to LXC
and can change at any point.

> The python bindings are typically very close to the C API except for the part where it exports
proper objects instead of structs.

The documentation is still kind of lacking though (will try and improve it where possible).

**Comment by Kentoseth** [comment]
My first goal when starting to work with the API was finding a command to show/return containers that exist on the system, but it seems that the [lxccontainer.h](https://github.com/lxc/lxc/blob/master/src/lxc/lxccontainer.h) was/is lacking this for some reason.

**Comment by stgraber** [comment]
with python3-lxc lxc.list_containers() or possibly lxc.list_containers(as_object=True)

As with any python module, running "help(lxc)" will list every API function available.

**Comment by Kentoseth** [comment]
Thank you.

I will attempt to improve the docs.

**Comment by Kentoseth** [comment]
I proceeded to improve the: https://linuxcontainers.org/ website.

However, when I was planning on adding the output of ```help(lxc)``` to the website, I stumbled across this:

https://linuxcontainers.org/lxc/apidoc/index.html

More specifically, this:

https://linuxcontainers.org/lxc/apidoc/structlxc__container.html

Is the link above a more detailed output of what ```help(lxc)``` shows?

I want to clarify this before making a contribution that may be rejected.
